website/docs/reference/snapshot-configs.md

```diff
@@ -80,7 +80,22 @@ snapshots:
 
 <TabItem value="property-yaml">
 
-**Note:** Required snapshot properties _will not_ work when defined in `config` YAML blocks. We recommend that you define these in `dbt_project.yml` or a `config()` block within the snapshot `.sql` file.
```

Comment:
Hi @nataliefiann - do we need to put this behind a version block? This will be available in dbt-core 1.9 (or now for cloud customers)

Reply:
Hiya @graciegoheen 

Happy Monday, i've added the version block.

Kind Regards
Natalie